### ISSUE TITLE
Trouble importing version 2.1.0

### ISSUE THREAD

**Issue by kasperp** [marcofugaro/browserslist-to-esbuild#10]
Hello,

Firstly, I want to express my appreciation for the browserslist-to-esbuild package. It's been very useful in my projects.

Recently, I upgraded to version 2.1.0 and encountered an issue with the import statement in my vite.config.ts file. Here's how I'm using it:

```typescript
import browserslistToEsbuild from 'browserslist-to-esbuild';

export default defineConfig({
   build: {
    target: browserslistToEsbuild()
  },
});
```

After the upgrade, the import fails with the error: Cannot find module 'browserslist-to-esbuild' or its corresponding type declarations.

I'm not an expert with NPM packages, but I found that modifying your `package.json` as shown below resolves the issue:

```diff
{
  "name": "browserslist-to-esbuild",
  ...
  "type": "module",
- "exports": {
-   "types": "./src/index.d.ts",
-   "default": "./src/index.js"
- }
+ "exports": {
+   ".": "./src/index.js"
+ },
+ "types": "./src/index.d.ts",
   ...
}
```

I'm not sure if I should be importing the module differently or if the exports field in `package.json` needs to be specified differently. Could you please provide some guidance on this? I can help out with a PR with this changes if that helps.

Thank you for your time and assistance.

**Comment by marcofugaro** [comment]
Hello, please share your vite and typescript versions.

**Comment by kasperp** [comment]
Sorry - these are my versions.

```
"typescript": "5.3.3",
"vite": "5.0.11",
```

**Comment by idleberg** [comment]
I can confirm the issue @kasperp runs into and the fix works for me as well. I have seen other repositories that expose `exports.types`, but I'm not sure what this is used for (other than providing an import for `my-package/types`). This is undocumented in both, the [npm](https://docs.npmjs.com/cli/v10/configuring-npm/package-json) and [Node](https://nodejs.org/api/packages.html#package-entry-points) documentation.

As far as I know, a package manifest should always provide a `types` / `typings` key, no matter whether it supplies `exports.types` or not.

Example project using both, top-level `types` and `export.types`:

https://github.com/sveltejs/svelte/blob/main/packages/svelte/package.json#L7

**Comment by marcofugaro** [comment]
Ah yeah `export.types` is supposed to replace `types`, but we're not there yet.
Thanks for the fix!

**Comment by marcofugaro** [comment]
Fixed in `v2.1.1`
